scss/components/_menu.scss

```diff
@@ -218,7 +273,7 @@ $menu-border: $light-gray !default;
     }
   }
   @else if $position == top {
-    > li > a {
+    li a {
```

Comment:
Generally the aim of the whole PR was to cut down on specificity (`.menu > li > a` was a nightmare to change).
I think the way we are generating the classes, so the parent for that will be `.menu.icon-right` would make this specific enough, bit I'm open to suggestions or correction 😄